app/src/test/java/org/fossasia/openevent/app/core/auth/login/LoginViewModelTest.java

```diff
@@ -107,49 +107,12 @@ public void shouldShowLoginError() {
         inOrder.verify(progress).onChanged(false);
     }
 
-    @Test
-    public void shouldLoginAutomatically() {
-        Mockito.when(authModel.isLoggedIn()).thenReturn(true);
-
-        loginViewModel.getLoginStatus().observeForever(login);
-
-        verify(login).onChanged(true);
-    }
-
     @Test
     public void shouldNotLoginAutomatically() {
         Mockito.when(authModel.isLoggedIn()).thenReturn(false);
 
-        loginViewModel.getLoginStatus().observeForever(login);
-
-        verify(login, Mockito.never()).onChanged(true);
-    }
-
-    @Test
-    public void shouldAttachEmailAutomatically() {
```

Comment:
This should be tested?

Reply:
I am adding test for StartViewModel. I will include this test in it. In login screen if the email is changed, it transitions back to the Start screen as in EventBrite so email list is not required in Login Screen now.